R/qenv-constructor.R

```diff
@@ -11,8 +11,23 @@
 #'
 #' @details
 #'
-#' `qenv()` instantiates a `qenv` with an empty environment.
-#' Any changes must be made by evaluating code in it with `eval_code` or `within`, thereby ensuring reproducibility.
+#' `qenv()` instantiates a with an empty `qenv` environment.
+#'
+#' @section `qenv` characteristics:
+#'
+#' A `qenv` inherits from the `environment` class, behaves like an environment, and has the
+#' following characteristics:
+#'
+#' - `qenv` environment is locked, and data modification is only possible through the `eval_code()`
+#' and `within()` functions.
+#' - It stores metadata about the code used to create the data.
+#' - Is immutable which means that each code evaluation does not modify the original `qenv`
+#' environment directly. See the following code:
+#'   ```
```

Comment:
Minor markdown details

```suggestion
#'
#' - It stores metadata about the code used to create the data.
#' - Is immutable which means that each code evaluation does not modify the original `qenv`
#' environment directly. See the following code:
#'
#'   ```
```

Reply:
I don't understand what this change suppose to do

Reply:
It was adding empty line before list and before chunk of code, I see it renders great now, ignore it